src/Admin/PageAdmin.php

```diff
@@ -115,14 +109,11 @@ protected function alterNewInstance(object $object): void
             $slugs = explode('/', $this->getRequest()->get('url'));
             $slug = array_pop($slugs);
 
-            try {
-                $parent = $this->pageManager->getPageByUrl($site, implode('/', $slugs));
-            } catch (PageNotFoundException $e) {
-                try {
-                    $parent = $this->pageManager->getPageByUrl($site, '/');
-                } catch (PageNotFoundException $e) {
-                    throw new InternalErrorException('Unable to find the root url, please create a route with url = /');
-                }
+            $parent = $this->pageManager->getPageByUrl($site, implode('/', $slugs)) ??
```

Comment:
This wasn't working before, getPAgeByUrl does not throw exception, but returns null.